pkg/datapoint/datapoint.go

```diff
@@ -205,11 +205,17 @@ func (p Point) MarshalTrace() ([]byte, error) {
 		meta := make(map[string]any)
 		point := node.(Point)
 		typ := "data_point"
-		meta["value"] = point.Value.Print()
+		if point.Value == nil {
+			meta["value"] = nil
+		} else {
+			meta["value"] = point.Value.Print()
+		}
```

Comment:
Maybe we can just skip the value field in that case?

```suggestion
		if point.Value != nil {
			meta["value"] = point.Value.Print()
		}
```